source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -175,3 +176,13 @@ function modify_query_loop_block_query_vars( $query, $block ) {
 
 	return $query;
 }
+
+/**
+ * We should only return pages from searches within this theme.
+ *
+ * @return void
+ */
+function remove_pages_from_search() {
+    global $wp_post_types;
+    $wp_post_types['page']->exclude_from_search = true;
+}
```

Comment:
This should probably be done via `register_{$post_type}_post_type_args` instead of altering the global directly.

Reply:
> I can't see any reason why pages should be included in the search for the showcase.
> 
> Normally I'd suggest using a `pre_get_posts` filter, checking for `is_search()` and setting `post_type` to the set of types that should be searched... but setting the post_type `exclude_from_search` param seems a reasonable approach as well.

Yeah, that sounds better. Will update.

Reply:
Done [c98b97b](https://github.com/WordPress/wporg-showcase-2022/pull/50/commits/c98b97b9aaec60bb5a5e9df48818f1bb8451f37c)

Reply:
Do i need to check for `$query->is_main_query()`?

Reply:
Yeah, best to check for `is_main_query`. LGTM Steve.